checklists/new-child-theme.md

```diff
@@ -0,0 +1,50 @@
+## Repo setup
+
+- [ ] Create a new repository on Bitbucket under the `projectlargo` account.
+	- Don't create it with any new files, readme, or license file. You want it to be blank.
+
+On your machine, in a new directory, create a style.css file:
+
+- [ ] theme style.css with [appropriate metadata](https://codex.wordpress.org/Theme_Development#Theme_Stylesheet).
+	- The best way to do this is [find an existing child theme](https://bitbucket.org/projectlargo/) and copy from there.
+
+Then, initialize git and add the new Bitbucket repository as a remote, according to the instructions Bitbucket gave you.
+
+```
+git init
+git remote add origin <url>
+git add style.css
+git commit -m "Initial commit"
+git push -u origin master
+```
+
+## Adding it to umbrella
+
+In the umbrella:
+
+```
+cd wp-content/themes/
+git submodule add <repo url> <repo name>
+git submodule update --init --recursive
+git commit -m "added new child theme"
+```
+
+## Adding files
+
+- [ ] theme functions.php
```

Comment:
probably add a reference here to the child theme docs, seems weird we would mention functions.php but not the other things that might be added, probably either just remove everything below this point and link to the sample child theme docs or flesh this out quite a bit more